### ISSUE TITLE
Alphanums vs printables in user/main

### ISSUE THREAD

**Issue by akaplan521** [taritter/Warmup-Project-SWE#1]
We started using printables instead of alphanums to account for the decimal in floats but now the parser isn't recognizing the end of the value when given a quoted expression 

ex: 
input: genre == "Fantasy" and author == "Samantha Shannon"
parses as
[[['genre', '==', '"Fantasy" and author == "Samantha Shannon"']]]

**Comment by taritter** [comment]
I edited the csv and json and added decimals to all. Cuz wasn’t the issue
that there were no values that were 4.4 for our rating so it just took the
4 and found that? So maybe you don’t need to do printables?

**Comment by akaplan521** [comment]
I don’t think that was the problem I’m trying to edit the parse values directly but it’s not working but we can just look tmrw

**Comment by akaplan521** [comment]
i dont think its possible you guys :( 😔
